### ISSUE TITLE
Checking 'additional start gcode' in baseline print causes no gcode to be generated

### ISSUE THREAD

**Issue by IronMonkeyL255** [teachingtechYT/teachingtechYT.github.io#58]
Checking 'additional start gcode' in baseline print causes no gcode to be generated when pressing the button at the bottom of the page. I changed the bed size (250x210), ABL method (Prusa MK3) and retraction distance (1mm) and it generated the code fine, but it would not generate when I attempted to add a purge line at the beginning. I even tried it with the box checked and no code in there, and it generated fine after unchecking the box.

**Comment by vector76** [comment]
I think there could be a bug depending what code is entered.  I put a dummy command G123 XYZ and it choked trying to do coordinate manipulation on an X coordinate of "YZ".  What exactly did you enter for your start code?

**Comment by IronMonkeyL255** [comment]
I copy pasted from Prusaslicer, so there shouldn't be any issues with it.


;go outside print area
G1 Y-3.0 F1000.0
G1 Z0.4 F1000.0
; select extruder
Tc
; purge line
G1 X55.0 F2000.0
G1 Z0.3 F1000.0
G92 E0.0
G1 X240.0 E25.0 F2200.0
G1 Y-2.0 F1000.0
G1 X55.0 E25 F1400.0
G1 Z0.20 F1000.0
G1 X5.0 E4.0 F1000.0

It also didn't generate when there wasn't anything in the box.

I was able to get it in there manually, and it ran fine (except it needed a bit more purging because of the MMU2S). I also had to add the Tx tool select prompt.
